datasets/norwegian_ner/README.md

```diff
@@ -0,0 +1,137 @@
+---
+annotations_creators:
+- expert-generated
+language_creators:
+- crowdsourced
+languages:
+- sv
+licenses:
+- other-
+multilinguality:
+- monolingual
+size_categories:
+- 10K<n<100K
+source_datasets:
+- original
+task_categories:
+- other
+task_ids:
+- other-other-Named Entity Recognition
```

Comment:
could you specify named entity recognition here ?

Reply:
What should be specified more precisely? I don't understand sorry.